test/support/fixtures/arguments_schema.ex

```diff
@@ -62,6 +67,16 @@ defmodule Absinthe.Fixtures.ArgumentsSchema do
   end
 
   query do
+    field :entities, list_of(:string) do
+      arg :representations, non_null(list_of(non_null(:any)))
+
+      resolve fn args, _ ->
+        # TODO: args = %{representations: [%{}]}
+        IO.inspect(args)
```

Comment:
Not sure whats causing this but the fields inside the input_object are getting removed somewhere